src/site/content/en/_redirects.yaml

```diff
@@ -5,11 +5,6 @@ redirects:
 - from: /policies
   to: https://policies.google.com/
 
-# Newsletter
```

Comment:
I'm not sure if it's safe to remove this redirect. I think we have existing places where `web.dev/subscribe` has been published and if we remove it those will 404.

One option would be to keep the redirect as is for now, and if we do a dedicated newsletter sign-up page, change it to point at that.